skbase/testing/test_all_objects.py

```diff
@@ -518,7 +518,19 @@ class TestAllObjects(BaseFixtureGenerator, QuickTester):
     """Package level tests for BaseObjects."""
 
     def test_create_test_instance(self, object_class):
-        """Check first that create_test_instance logic works."""
+        """Check create_test_instance logic and basic constructor functionality.
+
+        create_test_instance and create_test_instances_and_names are the
+        key methods used to create test instances in testing.
+        If this test does not pass, validity of the other tests cannot be guaranteed.
+
+        Also tests inheritance and super call logic in the constructor.
+
+        Tests that:
+        * create_test_instance results in an instance of estimator_class
```

Comment:
I don't think this is going to render correctly in sphinx. I think that bulleted lists need a space between the list bullets and the text above or below

Reply:
So in this case you'd need something like:


```python
"""Some doc summary.

Some extended summary.

Tests that:

* first bullet
* second bullet
* third bullet
"""
````

Reply:
resolved in all instances